api/v1/clusterextension_types.go

```diff
@@ -482,6 +482,9 @@ type ClusterExtensionStatus struct {
 	// When Progressing is True and the Reason is Succeeded, the ClusterExtension is making progress towards a new state.
 	// When Progressing is True and the Reason is Retrying, the ClusterExtension has encountered an error that could be resolved on subsequent reconciliation attempts.
 	// When Progressing is False and the Reason is Blocked, the ClusterExtension has encountered an error that requires manual intervention for recovery.
+	// <opcon:experimental:description>
+	// When Progressing is True and Reason is RollingOut, the ClusterExtension has one or more ClusterExtensionRevisions in active roll out.
```

Comment:
It would be nice to report in the PR description the changes to `ClusterExtension` conditions, similar to what you did for `ClusterExtensionRevision`